packages/cli/package.json

```diff
@@ -30,10 +30,10 @@
     "access": "public"
   },
   "dependencies": {
-    "@rollup/plugin-commonjs": "^21.0.0",
-    "@rollup/plugin-node-resolve": "^13.0.0",
-    "@rollup/plugin-replace": "^2.3.4",
-    "@rollup/plugin-terser": "^0.1.0",
+    "@rollup/plugin-commonjs": "^25.0.0",
+    "@rollup/plugin-node-resolve": "^15.0.0",
```

Comment:
I _thought_ **"@rollup/plugin-node-resolve"** was now part of core rollup, but I don't see it in the [3.0.0 release notes](https://github.com/rollup/rollup/blob/master/CHANGELOG.md#300) so I guess I was wrong on that one.  But the release notes are comprehensive at least.  😇